model/wrapper.js

```diff
@@ -0,0 +1,44 @@
+var bcrypt = require('bcrypt')
```

Comment:
Since this wrapper is specific to authentication, would there be a use case in separating out different types of wrapper classes? Maybe we can have a wrapper folder? Something to think about.